.github/workflows/contract-tests.yml

```diff
@@ -147,51 +150,114 @@ jobs:
 
       - name: Test 4844
         run: yarn test:4844
-  test-e2e:
-    name: Test e2e
+  # test-e2e:
```

Comment:
Should these still be commented out?

Reply:
probably not, will create a pr to fix this later